docs/book/deploying-zenml/zenml-self-hosted/deploy-with-docker.md

```diff
@@ -266,6 +263,13 @@ The following secure headers environment variables are supported:
 * **ZENML\_SERVER\_SECURE\_HEADERS\_CACHE**: The `Cache-Control` HTTP header value. The default value is `no-store, no-cache, must-revalidate`.
 * **ZENML\_SERVER\_SECURE\_HEADERS\_PERMISSIONS**: The `Permissions-Policy` HTTP header value. The default value is `accelerometer=(), camera=(), geolocation=(), gyroscope=(), magnetometer=(), microphone=(), payment=(), usb=()`.
 
+If you prefer to activate the server automatically during the initial deployment and also automate the creation of the initial admin user account, this legacy behavior can be brought back by setting the following environment variables:
+
+* **ZENML\_SERVER\_AUTO\_CREATE\_DEFAULT\_USER**: Set this to `1` to automatically create the initial admin user account when the server is first deployed. Defaults to `0`.
+* **ZENML\_DEFAULT\_USER\_NAME**: The name of the initial admin user account created by the server on the first deployment, during database initialization. Defaults to `default`.
+* **ZENML\_DEFAULT\_USER\_PASSWORD**: The password to use for the initial admin user account. Defaults to an empty password value, if not set.
```

Comment:
Maybe I misread the whole thing, but it seems that Name and Password were whipped from everywhere, but left here. Is that right?

Reply:
Yes, we'll need to point people to this section in the docs if they also want to deploy the old dashboard instead of the new one. The old dashboard doesn't support the server activation flow.

Reply:
Ah, clear